File: Components/RepositoryComments/RepositoryCommentInfo.vb
Imports System
Imports System.Configuration
Imports System.Data
Imports System.Data.SqlClient
Imports System.Data.SqlTypes
Imports System.Reflection
Imports System.Xml
Imports DotNetNuke
Imports DotNetNuke.Common.Utilities
Imports DotNetNuke.Common.Globals
Imports DotNetNuke.Services.Search

Namespace DotNetNuke.Modules.Repository
    Public Class RepositoryCommentInfo

        Private _ItemID As Integer
        Private _ObjectId As Integer
        Private _CreatedByUser As String
        Private _CreatedDate As Date
        Private _Comment As String

        ' initialization
        Public Sub New()
        End Sub

        ' public properties
        Public Property ItemId() As Integer
            Get
                Return _ItemID
            End Get
            Set(ByVal Value As Integer)
                _ItemID = Value
            End Set
        End Property

        Public Property ObjectId() As Integer
            Get
                Return _ObjectId
            End Get
            Set(ByVal Value As Integer)
                _ObjectId = Value
            End Set
        End Property

        Public Property CreatedByUser() As String
            Get
                Return _CreatedByUser
            End Get
            Set(ByVal Value As String)
                _CreatedByUser = Value
            End Set
        End Property

        Public Property CreatedDate() As Date
            Get
                Return _CreatedDate
            End Get
            Set(ByVal Value As Date)
                _CreatedDate = Value
            End Set
        End Property

        Public Property Comment() As String
            Get
                Return _Comment
            End Get
            Set(ByVal Value As String)
                _Comment = Value
            End Set
        End Property

    End Class
End Namespace

File: Components/RepositoryComments/RepositoryCommentInfo.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Diagnostics;
using System.Drawing;
using System.Linq;
using System.Configuration;
using System.Data.SqlClient;
using System.Data.SqlTypes;
using System.Reflection;
using System.Xml;
using DotNetNuke;
using DotNetNuke.Common.Utilities;
using DotNetNuke.Common;
using DotNetNuke.Services.Search;

namespace DotNetNuke.Modules.Repository
{
	public class RepositoryCommentInfo
	{

		private int _ItemID;
		private int _ObjectId;
		private string _CreatedByUser;
		private System.DateTime _CreatedDate;

		private string _Comment;
		// initialization
		public RepositoryCommentInfo()
		{
		}

		// public properties
		public int ItemId {
			get { return _ItemID; }
			set { _ItemID = value; }
		}

		public int ObjectId {
			get { return _ObjectId; }
			set { _ObjectId = value; }
		}

		public string CreatedByUser {
			get { return _CreatedByUser; }
			set { _CreatedByUser = value; }
		}

		public System.DateTime CreatedDate {
			get { return _CreatedDate; }
			set { _CreatedDate = value; }
		}

		public string Comment {
			get { return _Comment; }
			set { _Comment = value; }
		}

	}
}
